sympy/integrals/meijerint.py

```diff
@@ -1247,7 +1247,7 @@ def _int0oo(g1, g2, x):
     >>> from sympy.integrals.meijerint import _int0oo
     >>> from sympy.abc import s, t, m
     >>> from sympy import meijerg, S
-    >>> g1 = meijerg([], [], [-S(1)/2, 0], [], s**2*t/4)
+    >>> g1 = meijerg([], [], [-S.Half, 0], [], s**2*t/4)
```

Comment:
Perhaps it is better to leave docstring examples unchanged.

Reply:
Yes, that is the idea. But I might have missed or pressed "Replace all"... Thanks for finding it!